features/step_definitions/book_index_page_steps.rb

```diff
@@ -0,0 +1,30 @@
+Given(/^there are (\d+) books in the database$/) do |number|
+  FactoryGirl.create_list(:book, 100)
+end
+
+Then(/^I see a list of books in the database$/) do
+  expect(Book.all.ids).to include (1...100)
+end
+
+Then(/^the books are ordered by published date$/) do
+  expect(Book.first).to eq Book.order(:published_date).first
+  expect(Book.last).to eq Book.order(:published_date).last
+end
+
+Then(/^the list of (\d+) books are paginated in pages of (\d+) books per page$/) do |number_of_books, number_of_pages|
```

Comment:
Unused block argument - `number_of_books`. You can omit all the arguments if you don't care about them.<br>Unused block argument - `number_of_pages`. You can omit all the arguments if you don't care about them.<br>Line is too long. [118/100]